### PR TITLE
Add compliance and QSG docs

### PR DESCRIPTION
**Pull request by sherenewso2**
## Purpose
Add corrected .md files for QSG and compliance docs.

### CODE REVIEW

File: en/docs/Compliance/general-Data-Protection-Regulation.md [hunk 1]
```diff
@@ -35,34 +16,30 @@ Server. This section outlines the major GDPR related features that WSO2
...
 
-For more resources on GDPR, see the white papers, case studies, solution
-briefs, webinars, and talks published on our [WSO2 GDPR
-homepage](https://wso2.com/solutions/regulatory-compliance/gdpr/) . You
-can also find the original GDPR legal text
-[here](http://eur-lex.europa.eu/legal-content/en/TXT/?uri=CELEX%3A32016R0679)
-.
+	  For more resources on GDPR, see the white papers, case studies, solution
+	  briefs, webinars, and talks published on our [WSO2 GDPR
+	  homepage](https://wso2.com/solutions/regulatory-compliance/gdpr/) . You
```
Comment (YvonneW) [inline en/docs/Compliance/general-Data-Protection-Regulation.md]: An extra space exists just before the period.

File: en/docs/Compliance/general-Data-Protection-Regulation.md [hunk 2]
```diff
@@ -181,7 +152,7 @@ the scope of this document.
 
 For more information on the topic, refer [Removing References to Deleted
```
Comment (YvonneW) [inline en/docs/Compliance/general-Data-Protection-Regulation.md]: refer --> see

File: en/docs/Compliance/general-Data-Protection-Regulation.md [hunk 3]
```diff
@@ -197,44 +168,37 @@ Following features are supported as part of Self-care User Portal:
...
-    **
+<br/>
 -   **The right to rectification -** Individuals can rectify incorrect
     data on their user profiles by themselves by logging into Self-care
     User Portal.  
-    **  
-    **
+<br/>
 -   **The right to restrict processing -** Individuals can make
     restrictions on their user profiles by themselves by logging into
```
Comment (YvonneW) [inline en/docs/Compliance/general-Data-Protection-Regulation.md]: logging into --> logging in to

File: en/docs/Compliance/general-Data-Protection-Regulation.md (hunk 1 above)
Comment (sherenewso2) [reply en/docs/Compliance/general-Data-Protection-Regulation.md]: Fixed with commit 895dc3b5173232dd1a2cd561fb9d5b264bb096bb

File: en/docs/Compliance/general-Data-Protection-Regulation.md (hunk 2 above)
Comment (sherenewso2) [reply en/docs/Compliance/general-Data-Protection-Regulation.md]: Fixed with commit 895dc3b5173232dd1a2cd561fb9d5b264bb096bb

File: en/docs/Compliance/general-Data-Protection-Regulation.md (hunk 3 above)
Comment (sherenewso2) [reply en/docs/Compliance/general-Data-Protection-Regulation.md]: Fixed with commit 895dc3b5173232dd1a2cd561fb9d5b264bb096bb

File: en/docs/Compliance/general-Data-Protection-Regulation.md [hunk 4]
```diff
@@ -292,6 +256,6 @@ profile data from individuals. Access control features supported in WSO2
 IS  such as role-based access control (RBAC), attribute-based access
 control can be used to cater this requirement.
 
-For more information on Role-based Access control, Attribute-based
-Access Control, and XACML, refer [Access Control and
-Entitlement](_Access_Control_and_Entitlement_Management_) page.
+!!! Info
+	  For more information on Role-based Access control, Attribute-based
+	  Access Control, and XACML, refer [Access Control and Entitlement](/key-concepts/access-control-and-entitlement-management) page.
```
[the remark this answers is not in the document]
Comment (sherenewso2) [reply en/docs/Compliance/general-Data-Protection-Regulation.md]: Fixed with commit 895dc3b

**Pull request merged**